src/components/tools/table-tool/editable-header-cell.tsx

```diff
@@ -37,7 +39,8 @@ export const EditableHeaderCell: React.FC<IProps> = ({ column: _column }) => {
   const handleClose = (accept: boolean) => {
     onEndHeaderCellEdit?.(accept ? nameValue : undefined);
   };
-  const style: React.CSSProperties = { width: column.width };
+  const ehcStyle: React.CSSProperties = { height, lineHeight: `${kCellLineHeight}px` };
```

Comment:
I don't think it's necessary, so I moved it into `.scss` here and in a few other locations. It's a little awkward though, because I'm using that constant in the javascript to measure the size the css will eventually render to, so that number is now in both the javascript and css and will need to be changed in both places if it changes again.

Reply:
Yes, that is an argument for keeping it in one place, so I leave it to your discretion which is better in this case.